spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -463,6 +463,9 @@ def getRegistrationStackSh(saltEnabled):
         elif [ -f /etc/redhat-release ]; then
             grep -v '^#' /etc/redhat-release | grep -q '\(Red Hat\)' && BASE="res"
             VERSION=`grep -v '^#' /etc/redhat-release | grep -Po '(?<=release )\d+'`
+        elif [ -f /etc/openEuler-release ]; then
```

Comment:
@vzhestkov @raulillo82 do we **really** need this?

The bootstrap with the script was already working, right?

Wasn't it using `/etc/os-release`?

Reply:
While this is clarified, I confirm that the bootstrap script still works properly.

Reply:
ACK, do not remove this for now. Let's clarify what we are going to do about salt itself :-)